internal/app/query/query.go

```diff
@@ -15,11 +15,59 @@
 package query
 
 import (
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
 	"google.golang.org/grpc"
 	"open-match.dev/open-match/internal/appmain"
+	"open-match.dev/open-match/internal/telemetry"
 	"open-match.dev/open-match/pkg/pb"
 )
 
+var (
+	ticketsPerQuery     = stats.Int64("openmatch.dev/query/tickets_per_query", "Number of tickets per query", stats.UnitDimensionless)
+	cacheTotalItems     = stats.Int64("openmatch.dev/query/total_cache_items", "Total number of tickets query service cached", stats.UnitDimensionless)
+	cacheDeltaItems     = stats.Int64("openmatch.dev/query/delta_cache_items", "Number of tickets added by query service in the last update", stats.UnitDimensionless)
+	cacheWaitingQueries = stats.Int64("openmatch.dev/query/waiting_queries", "Number of waiting queries in the last update", stats.UnitDimensionless)
+	cacheUpdateLatency  = stats.Float64("openmatch.dev/query/update_latency", "Time elapsed of each query cache update", stats.UnitMilliseconds)
+
+	ticketsPerQueryView = &view.View{
+		Measure:     ticketsPerQuery,
+		Name:        "openmatch.dev/query/tickets_per_query",
+		Description: "Tickets per query",
+		Aggregation: telemetry.DefaultMillisecondsDistribution,
+	}
+	cacheTotalItemsView = &view.View{
+		Measure:     cacheTotalItems,
+		Name:        "openmatch.dev/query/total_cache_items",
+		Description: "Total number of tickets query service cached",
+		Aggregation: view.LastValue(),
+	}
+	cacheDeltaItemsView = &view.View{
+		Measure:     cacheDeltaItems,
+		Name:        "openmatch.dev/query/delta_cache_items",
+		Description: "Number of tickets cache delta items",
+		Aggregation: view.LastValue(),
```

Comment:
Delta can be calculated in prometheus with a query.  It's also not likely to be useful, maybe there's big updates, but most of the time I only do a small update, so this metric would look too low.  However it would be useful to know the rate of change, instead of the last value.  So maybe include that metric instead?

Reply:
Removed this metric.
rate of query can also be calculated via a query and is included in the next PR.

Reply:
It's not the rate of query calls which is interesting here, it's the rate of tickets being added to the cache.